plugins/scaffolder-node-test-utils/src/actions/mockActionContext.ts

```diff
@@ -43,7 +43,12 @@ export const createMockActionContext = <
     output: jest.fn(),
     createTemporaryDirectory: jest.fn(),
     input: {} as TActionInput,
-    checkpoint: jest.fn(),
+    async checkpoint<T extends JsonValue | void>(opts: {
```

Comment:
Shall we split this change out into a separate changeset?

Reply:
Created a separate change for this here: https://github.com/backstage/backstage/pull/28850